packages/iTwinUI-react/src/core/Tabs/Tabs.test.tsx

```diff
@@ -127,7 +128,7 @@ it('should render strings in HorizontalTab child component', () => {
   });
 });
 
-it('should add .iui-large if HorizontalTab has sublabel', () => {
+it('should add .iui-large if has sublabel', () => {
```

Comment:
This line would make a bit more sense with an object to replace "HorizontalTabs"; maybe "tab" or "it"

Reply:
Thanks, replaced. The new name is `should add .iui-large if tabs have sublabel`

Reply:
Looks good!